app/flatpak-builtins-build-export.c

```diff
@@ -628,6 +633,12 @@ validate_exports (GFile *export, GFile *files, const char *app_id, GError **erro
   if (!validate_service_file (service_file, activatable, files, app_id, error))
     return FALSE;
 
+  autostart_path = g_strconcat ("share/gnome/autostart/", app_id, ".desktop", NULL);
```

Comment:
`etc/xdg/autostart/` ?

Reply:
I guess this falls apart for exports since there isn't an `XDG_CONFIG_DIRS`...

Reply:
Yeah, i don't like hardcoding gnome in the generic flatpak paths.